app/uk/gov/hmrc/accessibilitystatementfrontend/views/StatementPage.scala.html

```diff
@@ -20,6 +20,7 @@
 @this(layout: Layout)
 
 @(statement: AccessibilityStatement)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@serviceAbsoluteUrl = {https://@statement.serviceDomain@statement.serviceUrl}
```

Comment:
Question on this one: in other places, I've been putting values like this within the class. Might be good to agree a consistent approach?

Reply:
You mean the AccessibilityStatement case class? I would vote for keeping rendering concerns away from the case class an in the template, so the case class is just about providing the raw data.